sdk/appcenter-crashes/src/androidTest/java/com/microsoft/appcenter/crashes/CrashesAndroidTest.java

```diff
@@ -628,7 +628,6 @@ public void run() {
         ArgumentCaptor<ManagedErrorLog> managedErrorLog = ArgumentCaptor.forClass(ManagedErrorLog.class);
         verify(mChannel, times(2)).enqueue(managedErrorLog.capture(), anyString(), eq(CRITICAL));
         assertNotNull(managedErrorLog.getValue());
-        assertEquals(mUserId, managedErrorLog.getValue().getUserId());
```

Comment:
Why it isn't true anymore?

Reply:
Because previously we got current userId (which was set after AC start) and now we get this data from the file with device history.